runtime-manager/modules/ROOT/pages/runtime-dep-serv-limits.adoc

```diff
@@ -23,7 +23,9 @@ The tested and recommended limits on the number of deployments in a hybrid deplo
 | Component | Tested/Recommended Limit
 | Maximum number of deployments per environment | 1000
 | Maximum number of deployments per hybrid target (not Runtime Fabric) | 200
-| Maximum deployment size | 200 MB
+| Maximum number of displayed properties per deployed app | 300
```

Comment:
Not just for deployed apps, even for apps yet to be deployed this limit exists. Maybe something on the lines of : "Maximum number of properties per application"